api/src/services/authorization.js

```diff
@@ -97,6 +97,10 @@ const allowedDoc = (docId, feed, { replicationKeys, contactsByDepth }) => {
   });
 };
 
+const alwaysAllowCreate = doc => {
+  return doc && doc.type && doc.type === 'feedback';
+};
```

Comment:
This is the only type of doc that offline users can create (excepting the ones that are already covered by filtered replication stuff), as far as I can tell.